receiver/linefile.go

```diff
@@ -58,16 +57,18 @@ func (lf *LineFile) read() error {
 		}
 	}
 	if err := scanner.Err(); err != nil {
-		lfLog.WithError(err).Error("Error reading file")
-		return skogul.Error{Reason: "Error reading file"}
+		return fmt.Errorf("unable to scan file: %w", err)
 	}
 	return nil
 }
 
 // Start never returns.
 func (lf *LineFile) Start() error {
 	for {
-		lf.read()
+		err := lf.read()
+		if err != nil {
+			lfLog.WithError(err).Error("Unable to read file")
+		}
```

Comment:
Good catch.

Not that it really matters here, and it's more of a preference, but it is possible to write this a bit more concise if the data (here: none) and error is not used further on:
```golang
if err := lf.read(); err != nil {
        // body
}
```